### DISCUSSION TITLE
[Feature Request] [Small fix/personalization option] make clock only show hour and minute

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#374] [Feature Requests]
please add the ability to change the clock to only show minutes and hour, as windows 10 doesn't have the option to show seconds, unless if showing seconds is your personal preference.

**Comment by DustinBrett** [comment]
If making your own fork the clock format can be changed in the code. Windows 10 actually can show seconds (via `ShowSecondsInSystemClock`) and I have had this on for many years. At some point in the future I plan to have a control panel and at that time I will have all settings customizable.

https://winhelponline.com/blog/show-seconds-taskbar-clock-showsecondsinsystemclock/

**Reply by CorruptionVR** [reply]
oh, ok

**Reply by DustinBrett** [reply]
But this can stay open as a feature request for when I have the control panel. I will add it one day.

**Reply by CorruptionVR** [reply]
also, a little off topic, but are you also planning on making the taskbar taller, because currently it is the height of when the "use small taskbar buttons" setting is enabled.

**Reply by DustinBrett** [reply]
Ya that is another of my preferences. Indeed it's based on the "use small" look. Again once I have a control panel I do plan to put all these things I have currently as my preference and allow them to be changed. But as it is now, daedalOS mirrors how my personal version of Windows 10 is setup without any mods to Windows itself beyond registry values.

Also the taskbar will be a bit trickier because when it comes to some of the looks they are hardcoded in a non-dynamic way where more code would need to change.

I also have a long term plan to redo the entire styling/theming part to be truly based on a single source of truth theme file, which would allow much more customization as well.

But these things are a few years off possibly. I do hope I can keep going on this project for a long time to come and eventually have every single thing anyone could ask for.
